crates/aranya-daemon/src/api.rs

```diff
@@ -388,7 +428,36 @@ impl DaemonApi for Api {
             .await
             .context("unable to create team")?;
         debug!(?graph_id);
-        Ok(graph_id.into_id().into())
+
+        let seed = match *self.quic.lock().await {
+            Some(ref mut quic_data) => {
+                let seed = QuicSyncSeed::<CS>::new(&mut Rng);
+
+                insert_seed(&mut quic_data.engine, &mut quic_data.store, seed.clone())
+                    .context("could not insert seed into keystore")?;
+
+                let psk = seed.gen_psk()?;
+                // Send PSK update to the key stores
```

Comment:
How many key stores are there? If there's only one, this should be singular:
```suggestion
                // Send PSK update to the key store
```

Reply:
"key stores" is an ambiguous term here but there are 2 which refer to the `ClientPresharedKeys` and `ServerPresharedKeys` types